lib/config.js

```diff
@@ -92,6 +93,6 @@ config.wallet = {
 };
 
 function _getReferenceId(name) {
-  return `${encodeURIComponent(window.location.hostname)}:` +
-    `${encodeURIComponent(name)}`;
+  const {hostname} = new URL('https://localhost:38443');
+  return `${encodeURIComponent(hostname)}:${encodeURIComponent(name)}`;
 }
```

Comment:
Please remove `_getReferenceId`.

Reply:
Removed and fixed [8a2a19a](https://github.com/digitalbazaar/bedrock-web-wallet/pull/4/commits/8a2a19acdec3e6e7f426ac09aab8fe7cc1c6ef4c)